ci-operator/jobs/openshift/ci-tools/openshift-ci-tools-master-postsubmits.yaml

```diff
@@ -1,31 +1,5 @@
 postsubmits:
   openshift/ci-tools:
-  - agent: kubernetes
```

Comment:
Do we know why there were changes in this file? I assume it's a linter automatically reordering this?

Reply:
@spadgett yes this was automatic change.